.nvmrc

```diff
@@ -1 +1 @@
-22.11.0
```

Comment:
I think that we need to change the one in the `WORKSPACE` as well.

Reply:
Technically those serve different purposes and do not need to be in sync. In general the one in the workspace is the min version that is supported by the framework. Whlist the other in the on that is used to run as a base version for the workflows. 

//cc @devversion, thoughts?

Reply:
yeah, I agree with Alan in general, but I also don't know that we enforce this strictly & that there is much NodeJS code user-facing here. The rule seems more useful for CLI repo.

In either case, I think we can update this to align the versions